dm/ctl/master/operate_source.go

```diff
@@ -31,7 +31,11 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> [config-file ...] [--print-sample-config]",
+		// the leading space could provide more friendly terminal printing
+		// cobra.Command doesn't support multi-line usage well
+		Use: `operate-source create|update [config-file ...] [--print-sample-config] [flags]
+        operate-source stop [config-file | sourceID ...] [--print-sample-config] [flags]
```

Comment:
--print-sample-config cannot work with create/update/stop at the same time. But change to print-sample-config look strange. 😖

Reply:
seems `--print-sample-config` will overwrite the result of create/update/stop/show, I guess user will only use `--print-sample-config` when they don't know how to write a config file (thus they can't do a right source operation), so it's not confusing.

going to change usage to
```
operate-source <operate-type> [config-file | sourceID ...] [--print-sample-config]
```